docs/src/site/twiki/BacklogMetricEmitterService.twiki

```diff
@@ -0,0 +1,54 @@
+---+Back Log Emitter Service
+
+Back Log Emitter service keeps track of all the process which have missed an SLA for a particular time and calculates backlog. Once a notification comes that process has completed, it updates the backlog. This service keeps on sending data to Graphite for each process which is missing SLA.
```

Comment:
This service keeps sending data to Graphite for each process instance that misses the SLA.